components/canvas/ChoiceEdgeLabel.tsx

```diff
@@ -0,0 +1,83 @@
+import { useReactFlow } from "reactflow";
+import { useEffect, useRef, useState } from "react";
+import { Button, Icon } from "@equinor/eds-core-react";
+import { edit } from "@equinor/eds-icons";
+import styles from "./ChoiceEdge.module.scss";
+import { patchEdge } from "@/services/graphApi";
+import { useProjectId } from "@/hooks/useProjectId";
+
+type EdgeLabelProps = {
+  id: string;
+  labelText?: string;
+  selected: boolean;
+};
+export function EdgeLabel({ id, labelText, selected }: EdgeLabelProps) {
+  const { setEdges } = useReactFlow();
+  const projectId = useProjectId();
```

Comment:
This can be destructured so that you won't have to call projectId.projectId in the patchEdge param

Reply:
fixed